### PR TITLE
add caprese salad

### PR DESCRIPTION
**Pull request by rsrinet**
issue#32

### CODE REVIEW

File: salads/caprese-salad.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+Basil
+Olive Oil
+Salt
+Pepper
+
+Steps:
+Step 1: Slice a few ripe tomatoes
+Step 2: Make same size slices of fresh mozzarella
+Step 3: Layer slices of tomatoes and mozzarella on a serving platter
+Step 4: Season with salt and pepper, drizzle some olive oil
```
Comment (jonathanholder) [inline salads/caprese-salad.md:13]: What about adding the basil, would this be a good place?
```suggestion
Step 4: Season with salt, pepper & chopped basil, drizzle some olive oil

```

**Review by jonathanholder — COMMENTED** [review]
Sounds good! Where does the basil come in?

File: salads/caprese-salad.md (hunk 1 above)
Comment (rsrinet) [reply salads/caprese-salad.md]: Thanks

**Review by jonathanholder — APPROVED** [review]
Thanks!

**Pull request merged**